examples/migration/CreateCompleteCampaignAdWordsApiOnly.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example creates serach campaign with the help of Adwords Api
+
+This code example is the first in a series of code examples that shows how to create
+a Search campaign using the AdWords API, and then migrate it to Google Ads API one
+functionality at a time. See other examples in this directory for code examples in various
+stages of migration.
+
+This code example represents the initial state, where the AdWords API is used to create a
+campaign budget, a Search campaign, ad groups, keywords and expanded text ads. None of the
+functionality has yet been migrated to the Google Ads API.
+"""
+
+
+import datetime
+import uuid
+from googleads import adwords
+import urllib.parse
+# Number of ads being added/updated in this code example.
+NUMBER_OF_ADS = 5
+# The list of keywords being added in this code example.
+KEYWORDS_TO_ADD = ["mars cruise", "space hotel" ]
+
+
+def createCampaignBudget(client):
+    """Creates a new budget and returns the newly created budget id.
+    Args:
+        client: Adwords client
+    Returns:
+        Budget id of the newly created budget.
```

Comment:
Try to be specific about the type of the return value. In this case it should be a `str`